lib/sandbox/postman-legacy-interface.js

```diff
@@ -109,6 +112,38 @@ getRequestBody = function (request) {
 };
 
 /**
+ *
+ * @param  {Uniscope} scope
+ * @param  {Execution} execution
+ * @param  {Object} pmapi
+ *
+ * @todo Move getAssertionObject function from pmapi-setup-runner.js to a common place and reuse it here too
+ */
+raiseAssertionEvents = function (scope, execution, pmapi) {
+    var tests = scope._imports && scope._imports.tests,
+        assertionIndex = pmapi.test.index(),
+        assertionEventName = EXECUTION_ASSERTION_EVENT_BASE + execution.id,
+        assertions;
+
+    if (_.isEmpty(tests)) {
+        return;
+    }
+
+    assertions = _.map(tests, function (value, key) {
+        return {
```

Comment:
need an `@note` here and in setup-test-runner making developer aware of this duplication